x-pack/plugins/apm/public/components/app/ServiceMap/index.tsx

```diff
@@ -29,7 +29,8 @@ interface ServiceMapProps {
   serviceName?: string;
 }
 
-export function ServiceMap({ serviceName }: ServiceMapProps) {
+export const ServiceMap = ({ serviceName }: ServiceMapProps) => {
+  const theme = useContext(ThemeContext);
```

Comment:
Can you create a hook that also contains the right type
```suggestion
  const theme = useTheme();
```

You can add the hook to `x-pack/plugins/apm/public/hooks/useTheme.ts`:
```ts
import { useContext } from 'react';
import { ThemeContext } from 'styled-components';
import { EuiTheme } from '../../../observability/public';

export function useTheme(): EuiTheme {
  return useContext(ThemeContext);
}
```